<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmView</class>
 <widget class="QDialog" name="fmView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewPicture</string>
  </property>
  <widget class="QLabel" name="labelMain">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>821</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 246, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 250);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderScale">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>830</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>410</number>
   </property>
   <property name="maximum">
    <number>1640</number>
   </property>
   <property name="value">
    <number>821</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
